### ISSUE TITLE
Wrong result in WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned

### ISSUE THREAD

**Issue by marekdedic** [WordPress/WordPress-Coding-Standards#1839]
Hi,
I think I found a bug in the `WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned` sniff.

I reduced it to a minimal example:

`phpcs.xml`:
```xml
<?xml version="1.0"?>
<ruleset>
	<config name="installed_paths" value="vendor/wp-coding-standards/wpcs"/>
	<rule ref="WordPress"/>
</ruleset>
```

`test.php`:
```php
<?php
/**
 * Lorem.
 *
 * @package pkg
 */

/**
 * Lorem.
 */
function fun() {
	fun2(
		new MyClass(
			array(
				new MyClass2(
					array(
						'a' => '',
						'navigation_background_hover_color' => '',
					)
				),
			)
		)
	);
}
```

I expected to get a warning on the line `'a' => ''`, however, there is none. Moreover, when I add spaces before the `=>` so that it aligns with the next row, I get a warning that there should be 1 space but 33 were found.

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.3.11-1~deb10u1
| PHP_CodeSniffer version | 3.5.2
| WPCS version            | 2.2.0
| WPCS install type       | Composer project local

## Tested Against `develop` branch?
- [ ] I have verified the issue still exists in the `develop` branch of WPCS.

**Comment by jrfnl** [comment]
@marekdedic Thank you for reporting this. I've had a look and this is not a bug, but a configuration issue.

The sniff has a `maxColumn` property which is set to `60` in the `WordPress-Core` ruleset to avoid excessively long lines due to the arrow alignment.

The alignment operator `=>` on second line in your array is at a position > 60 and is therefore disregarded when determining what the arrow alignment should be.

You can see this if you add a third item like `'abc' => ''` to the array. In that case, you will get an error based on the arrow in this third item for the first `a` item.

For a full explanation and example code for how to change the property value, please have a read through the wiki section about it:
https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#array-alignment-maximum-column

**Comment by marekdedic** [comment]
Hi,
thanks for the quick response!

Yeah, changing this property alleviates the issue. However, I'm not sure why this limit even is there when there is no line length limit AFAIK?

**Comment by jrfnl** [comment]
> I'm not sure why this limit even is there when there is no line length limit AFAIK?

@marekdedic Well, there's some discussion about that (and making it stricter (shorter)) in #1761. That may give you the additional context you are looking for.

**Comment by jrfnl** [comment]
@marekdedic Considering you've by now probably updated your configuration, can this issue be closed ?

**Comment by marekdedic** [comment]
Hi, yes, it can :)

Thanks for the help
